package.json

```diff
@@ -16,42 +16,33 @@
+    "bootstrap": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
     "clean": "lerna run clean",
     "compile": "lerna run --stream compile",
     "lint": "lerna run lint",
     "publish": "node scripts/publish.js",
     "prepush": "npm run lint",
     "test": "lerna exec --concurrency 1 --stream --bail=false -- npm run test",
-    "test:unit":
-      "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit",
+    "test:unit": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit",
     "test:integration": "lerna run test:integration --concurrency 1 --stream",
-    "test:without-system-tests":
-      "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test",
-    "test:system-tests":
-      "lerna run test --scope system-tests --concurrency 1 --stream",
-    "coverage:system-tests":
-      "lerna run coverage --scope system-tests --concurrency 1 --stream",
-    "vscode:package":
-      "lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1 && node scripts/reformat-with-prettier",
+    "test:without-system-tests": "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test",
+    "test:system-tests": "lerna run test --scope system-tests --concurrency 1 --stream",
+    "coverage:system-tests": "lerna run coverage --scope system-tests --concurrency 1 --stream",
+    "vscode:package": "lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1 && node scripts/reformat-with-prettier",
     "vscode:sha256": "lerna run vscode:sha256 --concurrency 1",
     "vscode:publish": "lerna run vscode:publish --concurrency 1",
     "watch": "lerna run --parallel watch",
-    "eslint-check":
-      "eslint --print-config .eslintrc.json | eslint-config-prettier-check",
+    "eslint-check": "eslint --print-config .eslintrc.json | eslint-config-prettier-check",
     "reformat": "node scripts/reformat-with-prettier.js",
-    "snyk:test":
-      "lerna exec --bail=false --ignore salesforcedx-vscode -- snyk test --severity-threshold=medium --show-vulnerable-paths=false",
-    "snyk:monitor":
-      "lerna exec --bail=false --ignore salesforcedx-vscode -- snyk monitor --severity-threshold=medium --show-vulnerable-paths=false --org=vazexqi",
+    "snyk:test": "lerna exec --bail=false --ignore salesforcedx-vscode -- snyk test --severity-threshold=medium --show-vulnerable-paths=false",
+    "snyk:monitor": "lerna exec --bail=false --ignore salesforcedx-vscode -- snyk monitor --severity-threshold=medium --show-vulnerable-paths=false --org=vazexqi",
     "aggregateJUnit": "node scripts/aggregate-junit-xml.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/forcedotcom/salesforcedx-vscode.git"
   },
-  "license": "BSD-3-Clause"
+  "license": "BSD-3-Clause",
```

Comment:
We don't need to modify this file as part of this changes.

Reply:
It keeps automatically updating the formatting for me :confused: Is there a way to stop that from happening for all future changes too? I remember Alli had asked me to run `node scripts/reformat-with-prettier.js` is that something I should be running each time?

Reply:
If you need to, you can create a local git ignore file for this. The number of times I've had to discard changes to this file is nuts.

Reply:
We should do something about this file getting reformatted every time we run `npm install` but this was added because line 47 was added.

Reply:
Ah okay, I see the change you're referring to. Deleting it now, thanks.